x-pack/plugins/ml/public/alerting/anomaly_detection_rule/register_anomaly_detection_rule.tsx

```diff
@@ -116,29 +116,29 @@ export function registerAnomalyDetectionRule(
     defaultActionMessage: i18n.translate(
       'xpack.ml.alertTypes.anomalyDetection.defaultActionMessage',
       {
-        defaultMessage: `[\\{\\{rule.name\\}\\}] Elastic Stack Machine Learning Alert:
-- Job IDs: \\{\\{context.jobIds\\}\\}
-- Time: \\{\\{context.timestampIso8601\\}\\}
-- Anomaly score: \\{\\{context.score\\}\\}
+        defaultMessage: `['{{rule.name}}'] Elastic Stack Machine Learning Alert:
+- Job IDs: '{{context.jobIds}}'
+- Time: '{{context.timestampIso8601}}'
+- Anomaly score: '{{context.score}}'
 
-\\{\\{context.message\\}\\}
+'{{context.message}}'
 
-\\{\\{#context.topInfluencers.length\\}\\}
+'{{#context.topInfluencers.length}}'
   Top influencers:
-  \\{\\{#context.topInfluencers\\}\\}
-    \\{\\{influencer_field_name\\}\\} = \\{\\{influencer_field_value\\}\\} [\\{\\{score\\}\\}]
-  \\{\\{/context.topInfluencers\\}\\}
-\\{\\{/context.topInfluencers.length\\}\\}
+  '{{#context.topInfluencers}}'
+    '{{influencer_field_name}}' = '{{influencer_field_value}}' ['{{score}}']
+  '{{/context.topInfluencers}}'
+'{{/context.topInfluencers.length}}'
 
-\\{\\{#context.topRecords.length\\}\\}
+'{{#context.topRecords.length}}'
   Top records:
-  \\{\\{#context.topRecords\\}\\}
-    \\{\\{function\\}\\}(\\{\\{field_name\\}\\}) \\{\\{by_field_value\\}\\}\\{\\{over_field_value\\}\\}\\{\\{partition_field_value\\}\\} [\\{\\{score\\}\\}]. Typical: \\{\\{typical\\}\\}, Actual: \\{\\{actual\\}\\}
-  \\{\\{/context.topRecords\\}\\}
-\\{\\{/context.topRecords.length\\}\\}
+  '{{#context.topRecords}}'
+    '{{function}}'('{{field_name}}') '{{by_field_value}}''{{over_field_value}}''{{partition_field_value}}' ['{{score}}']. Typical: '{{typical}}', Actual: '{{actual}}'
+  '{{/context.topRecords}}'
+'{{/context.topRecords.length}}'
 
-\\{\\{! Replace kibanaBaseUrl if not configured in Kibana \\}\\}
-[Open in Anomaly Explorer](\\{\\{\\{kibanaBaseUrl\\}\\}\\}\\{\\{\\{context.anomalyExplorerUrl\\}\\}\\})
+'{{! Replace kibanaBaseUrl if not configured in Kibana }}'
+[Open in Anomaly Explorer]('{{\\{kibanaBaseUrl}}'\\}'{{\\{context.anomalyExplorerUrl}}'\\})
```

Comment:
<img width="733" alt="image" src="https://github.com/elastic/kibana/assets/5236598/44f8edcf-4b5e-4fd8-af81-fc91d5c7ce53">

The URL template is broken apparently

Reply:
Fixed and updated the test snapshots as well (only chagne is `FormattedMessage` now became `<Memo(MemoizedFormattedMessage)`)